evm/src/NttManager/ManagerBase.sol

```diff
@@ -482,7 +527,12 @@ abstract contract ManagerBase is
     }
 
     function _checkThresholdInvariants() internal view {
-        uint8 threshold = _getThresholdStorage().num;
+        _checkThresholdInvariants(_getThresholdStorage().num);
+    }
+
+    function _checkThresholdInvariants(
+        uint8 threshold
+    ) internal pure {
```

Comment:
I'm surprised the compiler doesn't complain at you for making this pure given we read state?

Reply:
That is curious. I changed it to view.

Reply:
Interesting. I changed this to view and now it is giving me this:

```
Warning (2018): Function state mutability can be restricted to pure
   --> src/NttManager/ManagerBase.sol:533:5:
    |
533 |     function _checkThresholdInvariants(
    |     ^ (Relevant source part starts here and spans across multiple lines).
```